### PR TITLE
refector: update version of dependencies [ECO-928]

### PR DESCRIPTION
**Pull request by karantalapalli**
refector: update version of dependencies [ECO-928]

### CODE REVIEW

**Review by Amitkanswal — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -4,14 +4,14 @@
   "author": "Contentstack",
   "version": "1.0.1",
```
Comment (abhishek305) [inline package.json:5]: update this to v1.0.2

File: package.json [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
-    "gatsby-plugin-sharp": "^2.10.0",
-    "gatsby-source-contentstack": "^2.2.12",
-    "gatsby-source-filesystem": "^2.7.0",
-    "gatsby-transformer-sharp": "^2.8.0",
+    "gatsby": "4.13.1",
+    "gatsby-plugin-manifest": "4.13.0",
+    "gatsby-plugin-offline": "5.13.0",
+    "gatsby-plugin-react-helmet": "5.13.0",
+    "gatsby-plugin-sharp": "4.13.0",
+    "gatsby-source-contentstack": "3.1.4",
```
Comment (abhishek305) [inline package.json:12]: update the source-plugin version to v4.0.0

**Review by abhishek305 — COMMENTED** [review]
@karantalapalli Please make these changes:

- update the source-plugin version to v4.0.0
- update the package.json version to 1.0.2

rest looks good

File: src/pages/blogs.js [hunk 3]
```diff
@@ -6,13 +6,13 @@ import { graphql } from 'gatsby';
 import '../style/css/style.css';
 import Layout from '../components/layout';
 
-function dateSetter(params) {
+/* function dateSetter(params) {
```
Comment (abhishek305) [inline src/pages/blogs.js:9]: Please remove the commented code if not required

File: src/templates/blog-post.js [hunk 4]
```diff
@@ -15,13 +15,13 @@ const { Carousel } = require('react-responsive-carousel');
 export default function Blogpost({ data }) {
   const result = data.contentstackBlogPosts;
 
-  function dateSetter(params) {
+  /* function dateSetter(params) {
```
Comment (abhishek305) [inline src/templates/blog-post.js:18]: remove comments

**Review by abhishek305 — COMMENTED** [review]
@karantalapalli Please remove the comments rest looks good

**Review by abhishek305 — APPROVED** [review]

**Pull request merged**